dpkgfiles/make_deb_package.py

```diff
@@ -112,6 +120,121 @@ def find(name, path):
+            'sha256sumamd64': '680b6e4b5d1267c8d055c845129c379f9ab47f1c5e0d589266b5432317c627c7',
+            'sha256sumi386': '680b6e4b5d1267c8d055c845129c379f9ab47f1c5e0d589266b5432317c627c7'
+    }
+    pythonTwistedConch = {
+            'url': 'http://mirrors.kernel.org/ubuntu/pool/main/t/twisted-conch/python-twisted-conch_11.1.0-1_all.deb',
+            'sha256sumamd64': '222edf6328c8a6de721a47e1743f0a552dc18b582ffb5592e35a46c1b92a301e',
+            'sha256sumi386': '222edf6328c8a6de721a47e1743f0a552dc18b582ffb5592e35a46c1b92a301e'
+    }
+    pythonTwistedLore = {
+            'url': 'http://mirrors.kernel.org/ubuntu/pool/main/t/twisted-lore/python-twisted-lore_11.1.0-1_all.deb',
+            'sha256sumamd64': '1a6412eb813f1043df25a29ac586385baa672b8b8d0b2d1463c76ae85ee3a7f3',
+            'sha256sumi386': '1a6412eb813f1043df25a29ac586385baa672b8b8d0b2d1463c76ae85ee3a7f3'
+    }
+    pythonTwistedMail = {
+            'url': 'http://mirrors.kernel.org/ubuntu/pool/main/t/twisted-mail/python-twisted-mail_11.1.0-1_all.deb',
+            'sha256sumamd64': '9ec78d0a31f1e7a6abb0b768da4cb17b7488e9213d682dc49da63f101783fb21',
+            'sha256sumi386': '9ec78d0a31f1e7a6abb0b768da4cb17b7488e9213d682dc49da63f101783fb21'
+    }
+    pythonTwistedNews = {
+            'url': 'http://mirrors.kernel.org/ubuntu/pool/main/t/twisted-news/python-twisted-news_11.1.0-1_all.deb',
+            'sha256sumamd64': '1f54c0051aeb62941d156a739c7d72ac824c64121da57148aba25f5eac4520b7',
+            'sha256sumi386': '1f54c0051aeb62941d156a739c7d72ac824c64121da57148aba25f5eac4520b7'
+    }
+    pythonTwistedRunner = {
+            'url': 'http://mirrors.kernel.org/ubuntu/pool/main/t/twisted-runner/python-twisted-runner_11.1.0-1_%s.deb' % arch,
+            'sha256sumamd64': '7ccdb539b0a9b7b7f0d50c4c64ba253d7b3f93a96404d1f2c445d8d2057916f7',
+            'sha256sumi386': '7ccdb539b0a9b7b7f0d50c4c64ba253d7b3f93a96404d1f2c445d8d2057916f7'
+    }
+    pythonTwistedWords = {
+            'url': 'http://mirrors.kernel.org/ubuntu/pool/main/t/twisted-words/python-twisted-words_11.1.0-1_all.deb',
+            'sha256sumamd64': 'e6a740c4937db0940e109636c773d8dc2049faaa856001480a501c1fcca7d8c8',
+            'sha256sumi386': 'e6a740c4937db0940e109636c773d8dc2049faaa856001480a501c1fcca7d8c8'
+    }
+    depends = [libqt4Designer, libqt4Help, libqt4Scripttools, libqt4Test,
+            libqtassistantclient4, libqtwebkit4, pythonPsutil, pythonPyasn1,
+            pythonQt4, pythonSip, pythonTwisted, pythonTwistedConch,
+            pythonTwistedLore, pythonTwistedMail, pythonTwistedNews,
+            pythonTwistedRunner, pythonTwistedWords]
+    for deb in depends:
+        fname = deb['url'].rsplit('/', 1)[1]
+        execAndWait('rm %s' % fname)
+        execAndWait('wget %s' % deb['url'])
+        if sha256sum(fname) == deb['sha256sum%s' % arch]:
+            print '%s was downloaded and verified successfully' % fname
+        else:
+            execAndWait('rm %s' % fname)
```

Comment:
I'd make this rm -f. On some systems, using rm will prompt and ask if you really want to delete something. rm -f forces the deletion.